src/utils/spark.py

```diff
@@ -10,6 +11,7 @@ def get_spark_testing_conf() -> SparkConf:
     Returns:
         SparkConf: SparkConf with settings for testing.
     """
+    hail_home = hl.__file__.replace("/__init__.py", "")
```

Comment:
To make more obvious that what we want here you can do this (Path has to be imported):
```suggestion
    hail_home = Path(hl.__file__).parent.as_posix()
```

I did something very similar yesterday, and added hail_home as a fixture. If you think it's a good idea, you can copy this https://github.com/opentargets/genetics_etl_python/pull/237/commits/6b737fc318646ed33990e530d6986f0cff4d9dc7

Reply:
Yeah, much nicer.